### PR TITLE
[import] authority matching

### PR DESCRIPTION
**Pull request by rastislav-chynoransky**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes https://jira.sng.sk/browse/WUCZ-16

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] I have updated the [CHANGELOG](../CHANGELOG.md)
- [ ] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: app/Importers/GmuRnlImporter.php [hunk 1]
```diff
@@ -37,9 +35,8 @@ class GmuRnlImporter extends AbstractImporter
 
     protected static $name = 'gml-rnl';
 
-    public function __construct(IFileRepository $repository, Translator $translator)
+    protected function init()
     {
-        parent::__construct($repository, $translator);
         $this->sanitizers[] = function ($value) {
             return empty_to_null($value);
```
Comment (eronisko) [inline app/Importers/GmuRnlImporter.php:41]: What's the reason for this change? What it was before seemed more intuitive to me.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file[^1].
 ## [Unreleased]
 ### Added
 - global word-break:keep-all
+- authority matching in importers
```
Comment (igor-kamil) [inline CHANGELOG.md:7]:
```suggestion
- authority matching in importers

```

File: app/Importers/AbstractImporter.php [hunk 3]
```diff
@@ -46,19 +49,25 @@ abstract class AbstractImporter implements IImporter {
...
-            ));
+            throw new LogicException(
+                sprintf('%s needs to define its $name static property', get_class($this))
+            );
         }
 
+        $this->authorityMatcher = $authorityMatcher;
         $this->repository = $repository;
         $this->translator = $translator;
+        $this->init();
```
Comment (igor-kamil) [inline app/Importers/AbstractImporter.php:66]: ok. now I see.

**Review by igor-kamil — APPROVED** [review]
LGTM
